### PR TITLE
Added WriteAttributes to REST API of leshan demo server

### PR DESCRIPTION
**Pull request by AButenko**
Signed-off-by: Anton Butenko <ant.butenko@gmail.com>

### CODE REVIEW

File: leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/servlet/ClientServlet.java [hunk 1]
```diff
@@ -235,12 +239,20 @@ protected void doPut(HttpServletRequest req, HttpServletResponse resp) throws Se
                 ContentFormat contentFormat = contentFormatParam != null
                         ? ContentFormat.fromName(contentFormatParam.toUpperCase())
                         : null;
-
-                // create & process request
-                LwM2mNode node = extractLwM2mNode(target, req);
-                WriteRequest request = new WriteRequest(Mode.REPLACE, contentFormat, target, node);
-                WriteResponse cResponse = server.send(registration, request, TIMEOUT);
-                processDeviceResponse(req, resp, cResponse);
+                if(path.length >= 3 && "attributes".equals(path[path.length - 1])){
```
Comment (sbernard31) [inline leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/servlet/ClientServlet.java]: we should move the contentFormat stuff in the `else` block as it does not concern write attributes ?

**Comment by AButenko** [comment]
> Thx @AButenko ! pray
> 
> Could you share/provide an example of the REST API or a easy command line to test it :)

I've used instructions and proposed API from issue #601 and here is the example:
http://127.0.0.1:8080/api/clients/client_name/6/0/0/attributes?pmin=30&pmax=120

File: leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/servlet/ClientServlet.java (hunk 1 above)
Comment (AButenko) [reply leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/servlet/ClientServlet.java]: you are right, missed that point, I'll update pull request

**Comment by AButenko** [comment]
Pull request updated.
Thx @sbernard31 for your review and attention to details :)

**Comment by AButenko** [comment]
Also it may be needed to update documentation here 
https://github.com/eclipse/leshan/wiki/LWM2M-Supported-features
if this pull request will be accepted.

**Comment by sbernard31** [comment]
- merged in master
- [LWM2M-Supported-features](https://github.com/eclipse/leshan/wiki/LWM2M-Supported-features) updated 

Thx again @AButenko

**Pull request merged**